tools/dependencies-report/src/test/resources/notices/Red Hat Universal Base Image minimal-NOTICE.txt

```diff
@@ -0,0 +1 @@
+TEST
```

Comment:
Is this an artifact of testing?

Reply:
It follows the pattern of other test notice files in the `src/test/resources` folder - see https://github.com/elastic/logstash/tree/master/tools/dependencies-report/src/test/resources/notices

Reply:
It replaces the text from the original notice with `TEST`, to aid with simplicity of testing, I believe